open-craft/opencraft
# BB-4549 Fix or document local Ocim DNS check issue

## Description

This PR turns off DNS validation for OCIM local devstack

## Supporting Information

[BB-4549](https://tasks.opencraft.com/browse/BB-4549)

## Testing Instructions

- Checkout to this branch `tinumide/fix_local_dns_check` in OCIM repository
- Spin up OCIM local devstack
- Go to registration UI at `localhost:3000` and check the availability of a domain name
- Domain name should be available
